Signal/src/AppDelegate.m

```diff
@@ -988,7 +1006,7 @@ - (void)application:(UIApplication *)application didReceiveLocalNotification:(UI
 - (void)application:(UIApplication *)application
     handleActionWithIdentifier:(NSString *)identifier
           forLocalNotification:(UILocalNotification *)notification
-             completionHandler:(void (^)())completionHandler
+             completionHandler:(void (^)(void))completionHandler
```

Comment:
I've seen these compiler warnings... but XCode (9.2?) was turning them into errors when I fixed this before.

Reply:
I'll revert this for now to avoid breaking anyones build.

Reply:
Actually, I kept the change after verifying it still builds on 9.2